src/itest/java/uk/gov/companieshouse/company/profile/CucumberFeaturesRunnerITest.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.companieshouse.company.profile;
+
+import io.cucumber.junit.Cucumber;
+import io.cucumber.junit.CucumberOptions;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(
+        features = "src/itest/resources/features",
+        plugin = {"pretty",
+                "json:target/cucumber-report.json"
+        },
+        glue = {"uk.gov.companieshouse.company.profile.steps",
+                "uk.gov.companieshouse.company.profile.configuration"})
```

Comment:
`glue` needs to contain both the steps and the cucumber spring configuration package paths